picasso/src/main/java/com/squareup/picasso/Picasso.java

```diff
@@ -471,14 +472,23 @@ public void shutdown() {
   }
 
   Request transformRequest(Request request) {
-    Request transformed = requestTransformer.transformRequest(request);
-    if (transformed == null) {
-      throw new IllegalStateException("Request transformer "
-          + requestTransformer.getClass().getCanonicalName()
-          + " returned null for "
-          + request);
+    if (requestTransformers == null) {
+      return request;
     }
```

Comment:
I wrestled between this and an empty ArrayList().  Effective Java says the latter, but Picasso has a history of conservative memory usage.  Side effect: test setup is cleaner.  Thoughts?

Reply:
Collections.emptyList() has no memory effect.

Reply:
oh, i see. in the builder. yeah, null seems fine.

Reply:
sorry, just realized this is Picasso.Builder. was thinking about the Request.Builder.
let's do a non-null list. you're just creating a shared Picasso instance.

Reply:
sorry about that.

Reply:
updated.